src/testFolder.ts

```diff
@@ -31,7 +31,8 @@ function testFolder<I, O, E>(suiteName: string,
                     const result = await target(test.input);
                     if (test.errorExpected) {
                         // if we want an error just reject again
-                        return Promise.reject(new Error(`Expected an error but instead resolved or returned with ${result}`));
```

Comment:
I would argue that this is a backwards incompatiable change and we shouldn't tag it with `1.0.3`. But also I don't really think the students will care.

Reply:
Yeah but people were getting timeouts because of it :(